mlflow/__init__.py

```diff
@@ -188,5 +189,6 @@
     "get_registry_uri",
     "set_registry_uri",
     "list_run_infos",
+    "infer_pip_requirements",
```

Comment:
I don't think we want to make this part of our fluent API. Can we import this API in `mlflow.models` for documentation / visibility purposes instead?

Reply:
Sure!